tests/components/device_tracker/test_entities.py

```diff
@@ -0,0 +1,110 @@
+"""Tests for fan platforms."""
+
+import unittest
+import pytest
+
+from homeassistant.components.device_tracker.config_entry import (
+    BaseTrackerEntity, ScannerEntity
+)
+from homeassistant.components.device_tracker.const import (
+    SOURCE_TYPE_ROUTER, ATTR_SOURCE_TYPE
+)
+from homeassistant.const import (
+    STATE_HOME,
+    STATE_NOT_HOME,
+    ATTR_BATTERY_LEVEL
+)
+
+
+class TestScannerEntity(unittest.TestCase):
```

Comment:
Can we instead add a scanner entity class and `async_setup_entry` to this test platform:
https://github.com/home-assistant/home-assistant/blob/dev/tests/testing_config/custom_components/test/device_tracker.py

If we mock a config entry we can probably forward config entry setup to the platform directly.

Then we can test setting up a config entry platform with the new scanner entity.

Please use standalone pytest test functions instead of test classes.

Reply:
So, I didn’t touch any of this in this pr and I’m already testing that by loading what I did create in ZHA ;) . I can do it though... :)

Reply:
OK, think I got it... LMK if I am off in left field or if this is what you meant.

Reply:
That's exactly what I meant. :+1: